tests/bgp/test_bgp_session.py

```diff
@@ -137,6 +137,9 @@ def test_bgp_session_interface_down(duthosts, rand_one_dut_hostname, fanouthosts
         pytest.skip("Warm Reboot is not supported on isolated topology")
 
     duthost = duthosts[rand_one_dut_hostname]
+    hwsku = duthost.facts['hwsku']
+    if hwsku == "Arista-7050CX3-32S-C28S4":
```

Comment:
this doesn't look to be scalable well... I wonder how other libra skus skips this test. @zjswhhh and @bingwang-ms do you know a better approach here?

I also prefer to use test marker, since it looks doable.

Reply:
@r12f This case can passed in latest run, remove the skip for now